src/libsyntax/feature_gate.rs

```diff
@@ -409,6 +409,12 @@ impl<'a> Context<'a> {
 
 pub fn emit_feature_err(diag: &SpanHandler, feature: &str, span: Span, explain: &str) {
     diag.span_err(span, explain);
+
+    // #23973: do not suggest `#![feature(...)]` if we are in beta/stable
```

Comment:
interesting, I guess it's a different workflow. I tend to reach for `blame`, which then points to this PR. *shrugs*